dandi/organize.py

```diff
@@ -9,13 +9,14 @@
 import os.path as op
 from pathlib import Path
 import re
+import uuid
 
 import numpy as np
 
 from . import get_logger
 from .exceptions import OrganizeImpossibleError
 from .pynwb_utils import get_neurodata_types_to_modalities_map, get_object_id
-from .utils import ensure_datetime, flattened, yaml_load
+from .utils import ensure_datetime, flattened, yaml_load, copy_file, move_file
```

Comment:
I can tell from the fact that the items here aren't alphabetized that you aren't using [pre-commit](https://github.com/pre-commit/pre-commit).  Please install pre-commit, run `pre-commit install` in your clone of the dandi-cli repository, and run `pre-commit run -a` to properly format all of your changes so far.